app/assets/stylesheets/forms.scss

```diff
@@ -67,3 +67,14 @@
 .vertical-drag {
   cursor: ns-resize;
 }
+
+.inset-text-field {
+  border-left: 5px solid #bcbcbc;
+  padding: 6px 0 6px 10px;
+  position: relative;
+  left: 25px;
```

Comment:
`left` is usually reserved for absolutely positioned elements, although it does work on others. Could this be achieved using `margin-left` instead? Or possibly even by extending the padding on line 73?

Reply:
Thanks Andy - fixed with `margin-left`